packages/eslint-config-gitcoin/index.js

```diff
@@ -17,21 +21,14 @@ module.exports = {
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-non-null-assertion": "off",
         "@typescript-eslint/no-empty-function": "off",
-        // "@typescript-eslint/no-unused-vars": [
```

Comment:
did you leave this commented out by accident?

Reply:
Yeah, i'll fix.

Reply:
ah yes I just noticed, weird maybe we're adding this rule somewhere else, I think it's useful to have